apps/web/app/modules/Profile/Profile.page.tsx

```diff
@@ -0,0 +1,121 @@
+import { useParams } from "@remix-run/react";
+import { useTranslation } from "react-i18next";
+
+import { useContentCreatorCourses } from "~/api/queries/useContentCreatorCourses";
+import { useUserDetails } from "~/api/queries/useUserDetails";
+import { ButtonGroup } from "~/components/ButtonGroup/ButtonGroup";
+import { Gravatar } from "~/components/Gravatar";
+import { Icon } from "~/components/Icon";
+import { PageWrapper } from "~/components/PageWrapper";
+import { Avatar } from "~/components/ui/avatar";
+import { Button } from "~/components/ui/button";
+import CourseCard from "~/modules/Dashboard/Courses/CourseCard";
+import { isAdminLike } from "~/utils/userRoles";
+
+import { ProfilePageBreadcrumbs } from "./ProfilePageBreadcrumbs";
+
+export default function ProfilePage() {
+  const { id = "" } = useParams();
+  const { data: userDetails } = useUserDetails(id);
+  const hasPermission = isAdminLike(userDetails?.role ?? "");
```

Comment:
```suggestion
  const { isAdminLike } = useUserRole()
  const hasPermission = isAdminLike
```
or even simplier:
```suggestion
  const { isAdminLike } = useUserRole()
```
(explanation below)

Reply:
We cannot use this hook here as useUserRole checks the current user while hasPermission is for the user who's profile we are viewing. Maybe I could change the name of the variable to make more sense

Reply:
I talked about that with @wielopolski, he said shared repo's aren't implemented yet in this project, and that they need to implement it ASAP, unless I misunderstood